src/content/deployment/subctl.en.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "subctl"
+weight: 10
+---
+
+`subctl` is a commandline utility designed to simplify the deployment and maintenance of
+submariner across your your clusters.
+
+### Synopsis
+
+`subctl [command] [--flags] ...`
+
+### Description
+
+The subctl program works with the [submariner-operator](https://github.com/submariner-io/submariner-operator)
+it helps automate the operator deployment process reducing the possibility of mistakes during
+the process.
+
+Subctl connects to your cluster(s) and performs the requested *command* operation. For the `join`
+operation it will start by deploying the submariner-operator.
+
+### Commands
+
+#### deploy-broker
+
+`subctl deploy-broker [flags]`
+
+The **deploy-broker** command will configure the cluster pointed out by the `--kubeconfig`
+(or `KUBECONFIG` env var) and the `--kubecontext` flag as the broker. It will install
+the necessary CRDs and the  `submariner-k8s-broker` namespace.
+
+It will generate the `broker-info.subm` file which can be used with the `join` command
+to connect clusters to the broker. This file contains the following details:
+
+ * Encryption PSK key
+ * Broker access details for subsequent `subctl` runs.
+ * Service discovery settings
+ * Globalnet settings
+
+##### deploy-broker flags
```

Comment:
Are you sure to mention all the flags? Perhaps just the major ones?

Reply:
I called it "detailed" guide ':D I could avoid -h though. But I wanted to document all.

Reply:
I'm removing the -h's at least, that doesn't help

Reply:
-h removed at least.